docs/design/jet/023-kafka-connect-source-connectors.md

```diff
@@ -189,6 +189,16 @@ There are also nice to have features:
 - Add benchmarks for Kafka connect source
 - Add Soak tests for Kafka connect source
 
+### Task Distribution
+**TaskMaxProcessorMetaSupplier**  evenly distributes the number of tasks specified in the properties input to cluster members.    
```

Comment:
Is there any reason why we would want to have more Kafka Connect tasks than we have total parallelism in the cluster? 
If not, then tasks.max should be at most equal to total parallelism.

I see that user may want to throttle the reading, but it can be also done via `setLocalParallelism`, although it's not 1:1 (you cannot force total parallelism 1), so `tasks.max` < totalParallelism makes sense.